### ISSUE TITLE
[Community Suggestion]: Subreddit

### ISSUE THREAD

**Issue by CrossbowDemon** [snowie2000/mactype#590]
A subreddit would be nice, post and pin update notices and changelogs as well as potentially attracting new developers for the project.

**Comment by snowie2000** [comment]
Sounds more realistic than a discord channel. 
@sammilucia Thoughts?

**Comment by sammilucia** [comment]
@CrossbowDemon @snowie2000 yes that sounds good! What worries me about Discord as a first step is the real-time nature (I'm just not online real-time .... does that make sense? 😜 probably not lol)

I'm certainly open to anyone who wants to help! 😊

**Comment by sammilucia** [comment]
By the way I don't have a Reddit account OR any karma, so I can't create a subreddit
